firehose-source/src/main/proto/events/envelope.proto

```diff
@@ -0,0 +1,50 @@
+package events;
```

Comment:
I see you've removed the protobuf build part from the pom. So is versioning those `.proto` files.

Could you add instructions on how to regenerate the `.java` files from the `.proto` files? In a README in `src/main/proto` for example? Or directly in the top README.
Exact command-line options would be welcome, so that we make sure we can rebuild 2 years from now ... :)